src/Libraries/CoreNodes/String.cs

```diff
@@ -310,15 +310,22 @@ public static int IndexOf(string str, string searchFor, bool ignoreCase = false)
                     ? StringComparison.InvariantCultureIgnoreCase
                     : StringComparison.InvariantCulture);
         }
-
-        public static int[] AllIndicesOf(string str, string searchFor, bool ignoreCase = false)
+        /// <summary>
+        /// Finds list of indices where sub-string appears inside a string.
+        /// </summary>
+        /// <param name="string">A string to search in.</param>
+        /// <param name="searchFor">Substring to search for.</param>
+        /// <param name="ignoreCase">Whether or not comparison takes case into account.</param>
+        /// <returns name="indexList">List of indices (integers) where substring is found</returns>
```

Comment:
Does this hide the return type information?

Reply:
@aparajit-pratap I changed the output name to be a little bit more descriptive, do you think I should revert the name to `int[]` ?
![image](https://user-images.githubusercontent.com/25138291/93861038-d02c4b80-fcb7-11ea-8971-f09c3e094951.png)

Reply:
@martinstacey Please do retain descriptive output ports. @aparajit-pratap, correct me if I'm wrong, but you just also wanted to add the `in[]` to it?

Reply:
@Amoursol that's correct, apologize for not being clearer.

Reply:
@martinstacey please add `int[]` to the description:
`type: int[], List of indices where substring is found`

Reply:
Addressed in 926c7cb